mail/dovecot2/Portfile

```diff
@@ -79,6 +79,33 @@ configure.args      --sysconfdir=${prefix}/etc \
 
 configure.cppflags  -I${prefix}/include/openssl
 
+subport dovecot2-sieve {
```

Comment:
You can't have a port defined in two places. If you add dovecot2-sieve as a subport here, you have to delete the old separate Portfile.

But is dovecot2-sieve really similar enough to dovecot2 for it to make sense as a subport? It seems to be overriding almost everything.

Reply:
my reason for suggesting doing it this way is, that -as far as I can see- dovecot2-sieve is dependent upon having dovecot2 installed -and- integrates completely into dovecot2

I did realise that the dovecot2-sieve will have to be removed - I just didn't know how to request that.

Reply:
That isn't a consideration for whether something should be a subport. The advantage of subports is that you can change a value in one place and have it inherited by all subports. If the subports are all using distinct values for things, that instead becomes a liability, and you're better off having separate portfiles.

Reply:
but that's also more-or-less what's happening : 
- dovecot2-sieve has to be rev-bumped whenever dovecot2 changes
- dovecot2-sieve has a direct reference to the dovecot2 version

Reply:
Well whatever makes maintenance easier for you. It just looked like there were a lot of options being set to different values in the subport.